Made a tag to help people fix an empty file JSON error. (#1393)
python-discord/bot: bot/resources/tags/empty_json.md

When creating a new JSON file you might run into the following error.

`JSONDecodeError: Expecting value: line 1 column 1 (char 0)`

In short, this means that your JSON is invalid in its current state. This could very well happen because the file is just new and empty.

> A JSON may never be completely empty. It is recommended to have at least one of the following in your json:

Reviewer: ```suggestion
JSON data must not be completely empty. It is recommended to have at least one of the following in your json:
```

```
object
array

> ```

Reviewer: No need to use a list when there are only two items. Furthermore, I don't like using the term "recommend" here. I suggest rephrasing it as "You probably want to use an object or an array."

> To resolve this issue, you create one of the above values in your JSON. It is very common to use `{}` to make an object, which is similar to a dictionary in python.

Reviewer: The first sentence would likely get removed anyway based on my other suggestions, but just in case...
```suggestion
To resolve this issue, create one of the above values in your JSON. It is very common to use `{}` to make an object, which is similar to a dictionary in python.
```
